README.md

```diff
@@ -193,13 +189,15 @@ After you deploy the Infinispan Operator, see [Deploying Strimzi](#deploying-str
 
 ### Deploying Strimzi
 
-[Strimzi](https://strimzi.io/) is an Operator and should be available in the [OperatorHub](https://operatorhub.io/operator/strimzi-kafka-operator). In the OpenShift Web Console, go to **Catalog** -> **OperatorHub** in the left menu and search for `Strimzi`.
+#### Kafka
+
+[Strimzi](https://strimzi.io/) is an Operator and should be available in the [OperatorHub](https://operatorhub.io/operator/strimzi-kafka-operator). In the OpenShift Web Console, go to **Catalog** -> **Installed Operators** in the left menu and search for `Strimzi`.
```

Comment:
Strimzi is not a operator but there is a Strimzi operator.
Strimzi is a kind of Kafka as a Service

Reply:
@xiezhang7 maybe:
```markdown
[Strimzi](https://strimzi.io/) Operator should be available in the [OperatorHub](https://operatorhub.io/operator/strimzi-kafka-operator).
```

?

Reply:
+1

Reply:
@spolti @ricardozanini the [README](https://github.com/kiegroup/kogito-cloud-operator/blob/70c7f8d13b1ed70cbd5f15b0bb21745e036a7958/README.md#kafka) is updated. Thanks.